cachito/web/migrations/versions/044548f3f83a_add_pip_dev_preview_flag.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
```

Comment:
Nitpick: I'm comparing this file with the `add_gomod_vendor_flag.py` one. That one says "Add the gomod-vendor flag" in the comment, could this have a similar message?

Out of curiosity, what tool (if any) did you use to generate this file?

Reply:
@chmeliik : https://github.com/release-engineering/cachito/blob/master/README.md#database-migrations

Reply:
Aaah :/ How did I forget that. Updated! Thanks